cypress/integration/pages/errorPages/pagehelper.js

```diff
@@ -0,0 +1,13 @@
+import config from '../../../support/config/services';
+
+const iterator = (filterCondition, tests, canonicalOnlyTests, ampOnlyTests) => {
+  Object.keys(config)
+    .filter(filterCondition)
+    .forEach(service => {
+      tests(service);
```

Comment:
With route passed to the helper you can run tests twice once with route passed in and once with route.amp

Reply:
also then nest the canon and amp tests into this to benefit from the same cy.visit, saving page views